### DISCUSSION TITLE
Synchronize PWAs?

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [filips123/PWAsForFirefox#272] [Q&A] [answered]
I work between a laptop, a tablet and a desktop PC - all of them Windows 10.

Is there a neat way to synchronize PWAs?

Thanks!

**Comment by filips123** [comment, marked as answer]
There is no built-in way to synchronize PWAs, but you could probably do this if you somehow sync PWAsForFirefox data directory (`%APPDATA%\FirefoxPWA` by default), specifically `profiles` and `config.json`.

For one-time sync, it shouldn't be too hard, as you can just copy that directory to the other two devices. Continous sync is harder and I don't know a proper solution. Maybe you can try to use OneDrive or something similar, but I don't know if this would break something or affect performance.

Also, you can set `FFPWA_USERDATA` environment variable to a custom directory where you want data to be saved instead of the default one. This may be useful when setting up sync.
